src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java

```diff
@@ -436,6 +437,7 @@ protected Object invokeDescribable(String symbol, Object _args) {
         NamedArgsAndClosure args = parseArgs(_args, metaStep!=null && metaStep.takesImplicitBlockArgument(),
                 UninstantiatedDescribable.ANONYMOUS_KEY, singleArgumentOnly, new HashSet<>());
         UninstantiatedDescribable ud = new UninstantiatedDescribable(symbol, null, args.namedArgs);
+        ud.setModel(symbolModel);
```

Comment:
Needed to set the `DescribableModel` in order for `hasSoleRequiredArgument` to work.